sematic/examples/lightning_resnet/README.md

```diff
@@ -0,0 +1,17 @@
+An example of using
+[PyTorch Lightning](https://pytorch-lightning.readthedocs.io/en/stable/index.html)
+to do distributed training of a model classifying images in the
+[CIFAR100 dataset](https://www.cs.toronto.edu/~kriz/cifar.html)
+using Resnet.
+
+This example can only be run on Kubernetes clusters set up with sufficiently powerful
+GPU nodes enabled. It makes use of Sematic's Ray integration, which is available in
+Sematic EE.
+
+The modeling code in `train_eval` is based on an example from
+[the pytorch lightning documentation](https://pytorch-lightning.readthedocs.io/en/stable/notebooks/lightning_examples/cifar10-baseline.html)
+It is available under the license [CC BY-SA](https://creativecommons.org/licenses/by-sa/2.0/)
+
+This pipeline also illustrates how you can test the end-to-end connections between
+steps in your pipeline to reduce the chance that your pipeline will fail
+mid-execution from a programming error. See [`test_pipeline.py`](./tests/test_pipeline.py).
```

Comment:
Please add screenshots of the pipeline graph and output in this doc, for eye candy.

Reply:
done

Reply:
Thank you! Please add a short sentence introducing what each image represents, or a figure legend to each image, so that people completely unfamiliar with the product get a better understanding of the Dashboard's capabilities.